spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -351,6 +351,44 @@ For example:
 ```
 stream create task-stream --definition "http --port=9000 | tasklaunchrequest-transform --uri=maven://org.springframework.cloud.task.app:timestamp-task:jar:1.2.0.RELEASE | task-launcher-local --maven.remote-repositories.repo1.url=http://repo.spring.io/libs-release"
 ```
+=== Launching a Composed Task From a Stream
```

Comment:
The composed-task and the feature description are listed after this section. I think it would have a better flow if we move the entire section of "stream launching tasks" after the composed-task (_see suggestions in the acceptance criteria_).